tests/Component/ResponseParser/FacetSetTest.php

```diff
@@ -397,13 +397,49 @@ public function testParseJsonFacet()
                     'numBuckets' => 12,
                     'buckets' => [],
                 ],
+                'price_range' => [
+                    'buckets' => [
+                        [
+                            'val' => 0,
+                            'count' => 7,
+                        ],
+                        [
+                            'val' => 100,
+                            'count' => 2,
+                        ],
+                        [
+                            'val' => 200,
+                            'count' => 1,
+                        ],
+                        [
+                            'val' => 300,
+                            'count' => 3,
+                        ],
+                        [
+                            'val' => 400,
+                            'count' => 1,
+                        ],
+                    ],
+                    'before' => [
+                        'count' => 0,
+                    ],
+                    'after' => [
+                        'count' => 2,
+                    ],
+                    'between' => [
+                        'count' => 14,
+                    ],
+                ],
             ],
         ];
 
+        $price_range = new \Solarium\Component\Facet\JsonRange(['local_key' => 'price_range', 'field' => 'price', 'start' => 1, 'end' => 300, 'gap' => 100, 'other' => 'all']);
```

Comment:
We should use the constant JsonRange::OTHER_ALL instead of 'all'.

Reply:
changed to constant